tests/readrows_test.go

```diff
@@ -241,7 +241,15 @@ func TestReadRows_Retry_PausedScan(t *testing.T) {
 	origReq := <-recorder
 	retryReq := <-recorder
 	if diff := cmp.Diff(clientReq, origReq.req, protocmp.Transform(), protocmp.IgnoreEmptyMessages()); diff != "" {
-		t.Errorf("diff found (-want +got):\n%s", diff)
+		origRows := origReq.req.GetRows()
+		// Check if rows or row ranges are present in requests
```

Comment:
Also can we update the comment to something like: "In nodejs, we add an empty row range to a full table scan request to simplify the resumption logic. This check verifies that when row ranges is present, it's also empty."

Reply:
Done.